upgrade_testing/command_line.py

```diff
@@ -124,19 +124,28 @@ def display_results(output_dir):
     test_suite = junitparser.TestSuite("Auto Upgrade Testing")
     output = []
     output.append("Pre script results:")
-    for test, result in results["pre_script_output"].items():
+    for test, result in results.get("pre_script_output", {}).items():
         output.append("\t{test}: {result}".format(test=test, result=result))
         test_case = junitparser.TestCase(test)
         if result == "FAIL":
-            test_case.result = junitparser.Failure("Test Failed")
+            test_case.result = [junitparser.Failure("Test Failed")]
         test_suite.add_testcase(test_case)
 
+    output.append("Autopkgtest result: ")
```

Comment:
This can be slightly confusing for people not knowing that autopkgtest is used for the upgrade tests. Maybe this message should be "Upgrade result" or something like that?

Reply:
Yep, fair enough